### PR TITLE
Update to latest SDK. Improve error handling.

### PR DESCRIPTION
**Pull request by austinkelleher**
- Update to latest SDK
- Improve error handling. New implementation does not error out early if a single whois call fails to resolve.
- Experimented with placing all relevant ingestion code in `index.ts`. This integration is very small and may serve as a good real-world starting example to ingest a single entity into our platform. There is a bit of cognitive overhead when we have 15 files to do something simple like this. Happy to change it to match our other patterns if we don't want to do this though. Thought it may make for a good blog post at some point as this integration was born from our accidental domain expiration. Having the code co-located in a single file may make it easier to digest in a short blog or something.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -1,25 +1,4 @@
...
-  coverageThreshold: {
-    global: {
-      statements: 100,
-      branches: 100,
-      functions: 100,
-      lines: 100,
-    },
-  },
+  ...require('@jupiterone/integration-sdk-dev-tools/config/jest'),
+  setupFiles: ['dotenv/config'],
```
Comment (aiwilliams) [inline jest.config.js]: 🎉

File: package.json [hunk 2]
```diff
@@ -1,72 +1,40 @@
 {
   "name": "@jupiterone/graph-whois",
   "version": "0.2.0",
-  "description": "A JupiterOne integration that ingests domain information from Whois service",
+  "description": "A JupiterOne managed integration for Google",
```
Comment (aiwilliams) [inline package.json:4]: 😜

File: package.json [hunk 3]
```diff
@@ -1,72 +1,40 @@
...
-    "lint:md": "remark .",
-    "lint:es": "eslint . --ext .ts --ignore-pattern dist",
-    "format": "yarn prettier --write '**/*.{ts,js,json,md}'",
-    "test": "jest"
+    "start": "j1-integration collect",
+    "graph": "j1-integration visualize",
+    "lint": "eslint . --cache --fix --ext .ts,.tsx",
+    "format": "prettier --write '**/*.{ts,js,json,css,md,yml}'",
+    "type-check": "tsc",
+    "test": "ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION=1 jest",
```
Comment (aiwilliams) [inline package.json]: 💯

**Review by aiwilliams — CHANGES_REQUESTED** [review]
Personally, I like the single-file approach for the simplest integration. The value of the `integration-template` is that it provides some guidance on organization by major area. I think all documentation can do as you've done here, and a blog post would do well to avoid any further complications.

File: package.json (hunk 3 above)
Comment (austinkelleher) [reply package.json]: Btw I moved this to `jest.setup.js` because it won't run on Windows in this format.

File: package.json (hunk 2 above)
Comment (austinkelleher) [reply package.json]: Thanks

**Review by aiwilliams — APPROVED** [review]

**Pull request merged**